torch/csrc/jit/tensorexpr/schedule.h

```diff
@@ -671,6 +671,29 @@ class TORCH_API Schedule {
   ScheduleNode* node_ = nullptr;
 };
 
+class TORCH_API LoopNest {
+  public:
+    LoopNest(const std::vector<Tensor*> tensors_to_compute);
+    Stmt* root_stmt() const {
+      return root_stmt_;
+    }
+
+    std::vector<Stmt*> getLoopStmtsFor(Tensor*) const;
+    Stmt* getLoopBodyFor(Tensor*) const;
+    std::unordered_map<Tensor*, Stmt*> tensor_to_stmt_;
+
+    void ComputeInline(Stmt* s);
+    void ApplyInlines();
+    void SplitWithTail(Stmt *s, int factor, Stmt** inner, Stmt **outer, Stmt **tail);
```

Comment:
Assume we still retain original LoopAxisTransform/SplitAxis... classes?

Reply:
No, `For` class has all the needed info, there is no need in those classes.